packages/terra-time-input/src/TimeInput.scss

```diff
@@ -174,9 +182,18 @@
     text-align: center;
     width: auto;
 
+    &:hover {
+      border-color: transparent;
+    }
+
     &.focused {
       background-color: var(--terra-time-input-desktop-focused-input-background, rgba(139, 194, 249, 0.3));
     }
+
+    &[disabled] {
```

Comment:
Are these style changes needed to replace the input field? If not, we should include these changes in the changelog.

Reply:
Added to the changelog